### ISSUE TITLE
Required CURIE missing from top hit

### ISSUE THREAD

**Issue by AlexanderPico** [wikipathways/BioThings_Explorer_PFOCR_prioritization#5]
https://arax.ncats.io/api/arax/v1.3/response/8a8007f6-d02d-4578-81e3-df041ef5541b

The results all should contain the MESH for Alzheimer's, but the first one definitely does not. At least not by eye or by its pfocr web page: https://pfocr.wikipathways.org/figures/PMC5541263__onc2016467f8.html.

First, double check the dropbox pfocr files that are actually used. Maybe it is in there?

Next, check the enrichment code block to see if it is being erroneously inserted somehow.

**Comment by ayushi-agrawal-gladstone** [comment]
Checked the PFOCR files, it is indeed the first scenario. It is in the figures_curie.csv file. (screenshot below).
<img width="443" alt="figure_curies_for_PMC5541263__onc2016467f8" src="https://user-images.githubusercontent.com/88406934/217967305-4e7b6901-f58b-4381-8659-c6e09282d2b9.png">

**Comment by AlexanderPico** [comment]
Interesting. But since it is **_not_** actually in the figure, the next thing to check is the script that generated that csv file. Somehow it is inserting it?  We should rerun this script anyways to produce a more efficient csv for our notebook use case. The read time on the two current csv files is a major bottleneck.  

For example:
* should filter out pathways with fewer than 5 CURIEs (not counting bioentities)
* a single table with figid, title, curie would suffice

**Comment by ayushi-agrawal-gladstone** [comment]
@AlexanderPico The csv code is working as expected. The MESH id for Alzheimer's disease (MESH:D000544) is in the [bte_chemicals_diseases_genes.ndjson](https://www.dropbox.com/s/bs04audl3g4frit/bte_chemicals_diseases_genes.ndjson?dl=0) that was created on April 5,2022 and this is the file that the csv code uses for generating the csv file. Screenshot below from the JSON file for the specific image file.

![Screenshot 2023-02-14 at 12 45 29 PM](https://user-images.githubusercontent.com/88406934/218859933-a49a3346-3c92-4cf1-9d01-aef6dc44f084.png)

The [notebook](https://github.com/wikipathways/pathway-figure-ocr/blob/master/notebooks/bte_export.ipynb) that generated this JSON file needs the latest PFOCR data URL as shown below. This URL does not work me. Does it work for you? If not, do you know file it is looking for?

```
latest_pfocr_data_url = ("https://www.dropbox.com/sh/nmm0pddamgmu32m/AACCNdFXJrpQRD-EZOPe73RCa?dl=1")
```

**Comment by AlexanderPico** [comment]
Looks like it should go to a folder. If so, then this one: https://www.dropbox.com/sh/4djki29ppaqoo0k/AAAJTGB0f4rD5cdKt4MRo70ja?dl=0  (not sure if dl=1 needs to be set?)

That folder contains `pfocr_diseases.rds` for example, which would contain the "ground truth" of whether Alzheimer's is in that figure:
 * If yes, then the "bug" is further up the pipeline
 * If no, then the "bug" is likely in the bte_export notebook

**Comment by ayushi-agrawal-gladstone** [comment]
**Observations:**
1. The ```pfocr_diseases.rds``` object from the latest release does not have Alzheimer's disease for the figure in question. 
2. I ran the bte_export notebook with the [latest PFOCR data](https://www.dropbox.com/sh/0m6jlf3uxfmi62p/AAAGoefq1AjVOlaiLnyhSS-Ka?dl=0) and the output JSON file did not have Alzheimer's disease for figure [PMC5541263__onc2016467f8.jpg](https://pfocr.wikipathways.org/figures/PMC5541263__onc2016467f8.html).  
3. The bte_export notebook was last run in April 2022. So, it would have used the data from release [20210515](https://www.dropbox.com/sh/p25r378rn1o4nha/AAAnrd48rJAYTOsVz_H9ivDLa?dl=0). 
4. The bte_export notebook reads in ```pfocr_figures_draft.rds```, ```pfocr_genes_draft.rds```, ```pfocr_chemicals_draft.rds``` and ```pfocr_diseases_draft.rds```. I could find all input object but not the ```pfocr_diseases_draft.rds``` object in the 20210515 release Dropbox folder. So, I cannot verify if there was an error in the input object that was used at that time. 
5. According to what I have found so far, the bte_export notebook is working fine with the data from the latest PFOCR release. The issue is probably because of the input data that was used when the notebook was last run. 

**Suggested solution:** 
Re-run the bte_export notebook with the latest PFOCR release data. We can either do this now or after the next release of PFOCR (which is soon?).
